client/app/services/query-result.js

```diff
@@ -224,6 +224,7 @@ function QueryResultService($resource, $timeout, $q) {
         let seriesName;
         let xValue = 0;
         const yValues = {};
+        let eValue = 0;
```

Comment:
@arikfr for some reason they do look fine even though with the errors = 0 they should look slightly different. I want to review this throughly on Monday, also I find very annoying that after I set an error/group by column I cannot clear it, so I want to add that feature as well.